includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -3928,7 +3928,7 @@ public function get_payment_method_ids_enabled_at_checkout( $order_id = null, $f
 
 				$skip_currency_check       = ! $force_currency_check && is_admin();
```

Comment:
I can't comment on it directly since it's not part of this PR, but I fumbled around a bit on the comments above. The `currency_is_checked` language is the opposite of `$skip_currency_check`. At first I thought our logic was inverted since if `$force_currency_check` and `is_admin()` are `false`, `$skip_currency_check` will also be `false`. I was seeing `1` in the comments above and was briefly confused about what should be happening here.

You don't have to do this, but I wouldn't be mad if `currency_is_checked` was switched to match the `$skip_currency_check` OR just changed to say "The currency WILL/WILL NOT be checked."